src/messages/Lifecycle.ts

```diff
@@ -53,6 +55,7 @@ const envDataDecoder = guard(
   object({
     locale: string,
     hostRootUrl: string,
+    registeredKeys: mixed,
```

Comment:
you should be able to give this a stronger guard.  Its just an array of strings, right?